app/controllers/api/v1/downloads_controller.rb

```diff
@@ -23,18 +23,14 @@ def show
   end
 
   def top
-    data = {
-      gems: Download.most_downloaded_today(50).map do |version, count|
-        [version.attributes, count]
-      end
-    }
-    respond_with_data(data)
+    render text: "This endpoint is not supported anymore", status: :not_found
   end
 
   def all
+    gems = GemDownload.where("version_id != 0").includes(:version).order(count: :desc).limit(50)
```

Comment:
I kind of like the approach of hiding these sorts of queries under method names in the models.
`most_downloaded_all_time` was maybe not the best name, but I still like the idea.